HPCC-16864 Download to CSV not working

miguelvazq opened the pull request:
Whenever submitting to download the selections to CSV nothing was occuring. Removed a 'header.join()' issue.

Signed-off by Miguel Vazquez <miguel.vazquez@lexisnexis.com>

miguelvazq commented:
@GordonSmith Please review.

HPCCSmoketest commented:
Automated Smoketest
Sha: cfa0c493c1c1d38b4b7b53282566ac3911cb13f3
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 91.11 seconds |

Install hpccsystems-platform-community_6.3.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|86 |86 |0 |0 |

HPCC Stop: OK
HPCC Uninstall: OK

GordonSmith reviewed (approved):
Looks good.

richardkchapman commented:
@miguelvazq Should this target 6.2.4 ?

miguelvazq commented:
@richardkchapman Yes.

Commit:
HPCC-16864 Download to CSV not working

Whenever submitting to download the selections to CSV nothing was occuring. Removed a 'header.join()' issue.

Signed-off by Miguel Vazquez <miguel.vazquez@lexisnexis.com>

miguelvazq commented:
@richardkchapman Target changed to 6.2.4

HPCCSmoketest commented:
Automated Smoketest
Sha: 2c4c85329b5824922f808b6e4fd438d500656142
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 90.874 seconds |

Install hpccsystems-platform-community_6.2.3-closedown0.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|88 |88 |0 |0 |

HPCC Stop: OK
HPCC Uninstall: OK
